aplib.net-demo/Assets/Scripts/Items/Items/TestItem2.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
```

Comment:
These are all unused.

Reply:
Testitems will be removed in final commit